docs/expressions.md

```diff
@@ -510,7 +510,20 @@ Group functions modify the OpenTSDB groups.
 ## addtags(set variantSet, group string) (seriesSet|numberSet)
 {: .exprFunc}
 
-Accepts a series and a set of tags to add to the set in `Key1=NewK1,Key2=NewK2` format. This is useful when you want to add series to set with merge and have tag collisions.
+Accepts a set and a set of tags to add to the set in `Key1=NewK1,Key2=NewK2` format. This is useful when you want to add series to set with merge and have tag collisions.
+
+## metatag(set variantSet, tagsCSV string, keyCSV string) (seriesSet|numberSet)
```

Comment:
What if this accepted a variable number of arguments, 1 tag each? And make encoding the new key optional perhaps (if the keys are the same). Thinking `metatag($m, host=alias, iface=desc, somethingElse)`